gdx-jnigen-gradle/build.gradle

```diff
@@ -0,0 +1,20 @@
+dependencies {
+    implementation project(":gdx-jnigen")
+    implementation gradleApi()
+}
+
+task sourcesJar(type: Jar) {
+    from sourceSets.main.allJava
+    archiveClassifier = 'sources'
+}
+
+publishing {
+    publications {
+        maven(MavenPublication) {
+            group "com.badlogicgames.gdx.gdx-jnigen"
+            artifactId = "com.badlogicgames.gdx.gdx-jnigen.gradle.plugin"
```

Comment:
Should be 
```
group "com.badlogicgames.gdx"
artifactId = "gdx-jnigen-gradle"
```

Reply:
Doing it this way requires a 
```
buildscript {
    dependencies {
        classpath "com.badlogicgames.gdx:gdx-jnigen-gradle:2.0.0"
    }
}
```

But that seems cleaner, so will change it over to that.